### PR TITLE
CI: Manage test tags in yaml

### PR DESCRIPTION
**Pull request by kmaehashi**

### CODE REVIEW

**Review by takagi — APPROVED** [review]

File: .pfnci/README.md [hunk 1]
```diff
@@ -10,12 +10,14 @@ This directory contains the test matrix definition, and a tool to generate test
 * [`schema.yaml`](schema.yml) defines all the possible values for each test axis (e.g., `cuda`, `python`, `numpy`), and constraints between them.
```
Comment (kmaehashi) [inline .pfnci/README.md:10]:
```suggestion
* [`schema.yaml`](schema.yaml) defines all the possible values for each test axis (e.g., `cuda`, `python`, `numpy`), and constraints between them.
```

File: .pfnci/README.md [hunk 2]
```diff
@@ -10,12 +10,14 @@ This directory contains the test matrix definition, and a tool to generate test
...
 * [`matrix.yaml`](matrix.yml) defines the configuration of each test matrix (e.g., `cupy.linux.cuda115`.)
```
Comment (kmaehashi) [inline .pfnci/README.md:11]:
```suggestion
* [`matrix.yaml`](matrix.yaml) defines the configuration of each test matrix (e.g., `cupy.linux.cuda115`.)
```

**Review by kmaehashi — COMMENTED** [review]
Fix dead links

**Comment by kmaehashi** [comment]
Merging as this gets approved and test failures are unrelated.

**Pull request merged**